modules/rules_mayhem/0.6.3/presubmit.yml

```diff
@@ -0,0 +1,12 @@
+bcr_test_module:
+  module_path: ".."
```

Comment:
`bcr_test_module` should used when you have a separate module that depends on rules_mayhem and tests its behavior. (For example, if your [`examples` directory](https://github.com/ForAllSecure/rules_mayhem/blob/main/examples/BUILD) had its own MODULE.bazel file.)

In your case, you can simply not use a test module by removing the first two lines of this file (and de-indenting all subsequent lines).

Reply:
I see, ok thanks. All tests for this rule are internal under the examples directory. I'll remove and update!